.circleci/workflows.yml

```diff
@@ -693,14 +683,9 @@ jobs:
             export IMAGE_TAG_LATEST=graviteeio.azurecr.io/am-management-ui:${TAG_LATEST}
             echo "export IMAGE_TAG=$IMAGE_TAG" >> $BASH_ENV
             echo "export IMAGE_TAG_LATEST=$IMAGE_TAG_LATEST" >> $BASH_ENV
```

Comment:
Not useful anymore if both commands are merged

Reply:
removed